### PR TITLE
Remove 'function' from docstrings

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Review by rfourquet — APPROVED** [review]

File: src/Rings/affine-algebra-homs.jl [hunk 1]
```diff
@@ -243,7 +243,7 @@ function (F::AlgHom)(p::U) where U <: Union{MPolyElem, MPolyQuoElem}
 end
 
 @doc Markdown.doc"""
-    function domain(F::AlgHom)
+    domain(F::AlgHom)
```
Comment (tthsqe12) [inline src/Rings/affine-algebra-homs.jl]: I don't think the point of a doc string is to prove that the function exists. If this is the case, I see no need whatsoever for doc string attached to `f(x)` that says "return f of x".

File: src/Rings/affine-algebra-homs.jl [hunk 2]
```diff
@@ -252,7 +252,7 @@ function domain(F::AlgHom)
 end
 
 @doc Markdown.doc"""
-    function codomain(F::AlgHom)
+    codomain(F::AlgHom)
```
Comment (tthsqe12) [inline src/Rings/affine-algebra-homs.jl]: Ditto.

File: src/Rings/affine-algebra-homs.jl (hunk 1 above)
Comment (fingolfin) [reply src/Rings/affine-algebra-homs.jl]: Sure. But I did not review which docstrings should go (many should) or should be added, this was entirely about fixing an obvious bug in multiple docstrings via a regex.

Comment (tthsqe12) [reply src/Rings/affine-algebra-homs.jl]: Ok. Glad to know we still should not be writing doc strings like this.

Comment (fingolfin) [reply src/Rings/affine-algebra-homs.jl]: I removed these and two other redundant docstrings for `domain` and `codomain`. Plenty more redundant docstrings remain, but I don't want to get lost in chasing those down right now.

**Comment by fingolfin** [comment]
@tthsqe12 OK to go? then please approve this so it's merged :-)

**Review by tthsqe12 — APPROVED** [review]

**Pull request merged**